### PR TITLE
Update install_prereqs with dreal and dependencies and patchelf

### PR DESCRIPTION
**Pull request by jamiesnape**
PTAL @stonier.

### CODE REVIEW

**Comment by stonier** [comment]
@soonho-tri dreal install rules failing?

**Comment by jamiesnape** [comment]
It's my problem, if they are. I will look into it.

**Comment by jamiesnape** [comment]
Of course, it would help if I actually installed the `.deb` instead of downloading and deleting it.

File: scripts/setup/mac/install_prereqs [hunk 1]
```diff
@@ -50,20 +50,22 @@ if [[ ! -f /usr/include/expat.h || ! -f /usr/include/zlib.h ]]; then
   exit 3
 fi
 
-brew tap dreal-deps/coinor
+brew tap dreal-deps/ibex
```
Comment (soonho-tri) [inline scripts/setup/mac/install_prereqs]: FYI, this line is not necessary (as we do not need `dreal-deps/coinor`). They are included by `brew tap dreal/dreal` and I think it's clearer in that way.

Comment (jamiesnape) [reply scripts/setup/mac/install_prereqs]: The behavior of fully-qualified names in `depends_on` is actually inconsistent. We stopped using them in `robotlocomotion/director` as they were causing issues.

Comment (soonho-tri) [reply scripts/setup/mac/install_prereqs]: I see. Could you drop a link to an upstream resource about it if you have any?

**Comment by jamiesnape** [comment]
Need to wait for the tonight's drake packages to build.

**Comment by jamiesnape** [comment]
CI is now happy.

**Pull request merged**